development/scripts/add-rn-startup-performance-tracking.js

```
@@ -0,0 +1,189 @@
+const $$perfStart_${varName} = typeof globalThis.nativePerformanceNow === 'function' ? globalThis.nativePerformanceNow() : Date.now();
+
+if (typeof globalThis === 'object') {
+  (globalThis as any).$$perfStart_${varName} = $$perfStart_${varName};
+}
+
+export {};
+`;
+
+    // Write ready file
+    fs.writeFileSync(readyFilePath, readyFileContent, 'utf-8');
+    stats.readyFilesCreated++;
```

ESLint

[error] Unary operator '++' used.

(no-plusplus)